### ISSUE TITLE
Cannot access Homeassistant locally 

### ISSUE THREAD

**Issue by Bytelink5616** [brenner-tobias/addon-cloudflared#384]
### The problem

I was able to access my Homeassistant locally using the url but since one of the last two updates I can still access HA remotelyhowever I cannot access it locally.  I am sure I have done someting or more to the point not done something so any help or advise would be appreciated

### What version of Cloudflared has the issue?

4.1.2

### What type of installation are you running?

Home Assistant OS

### Add-on YAML Configuration

```yaml
external_hostname: ha.bytelink.us
additional_hosts: []
```

### Steps to reproduce the issue

just try to access url from local network

**Comment by brenner-tobias** [comment]
How are you trying to access HA when in your home network? Using the internal IP of HA or also the external hostname?
Are you talking about accessing HA in the Smartphone app or on your PC in a web browser?

**Comment by brenner-tobias** [comment]
Unfortunately, the screens cannot be attached via email.
Not really sure what might cause a local connection to fail after setting-up the addon...
Are you using http or https when connecting? Did you change anything in the http integration in the HA config except adding the trusted proxies? Did the connection work before installing the add-on?

Maybe @elcajon has an idea?

**Comment by Bytelink5616** [comment]
Thank you for your responses.  I tried all ways to access the home assistant locally via the ip:8123 and hostname:8123 which I had added to the DNS for local resolution but still nothing.  I also tried both http and https but to no avail.  So I decided to restore the system from a backup when all worked fine.  Then when I tried to access the machine locally all was good and I was able to access HA again.  I then updated all the integrations addons and HACS one by one and checking for local access each time and now I have a current HA with all the updates and I can access the system remotely and locally.  I do not know what happened to the system previously but it is working fine now and I appreciate all your help and suggestions.  I am sorry to have bothered you and it was obviously coincidence that the problem seemed to appear after the addon update?  Once again thanks for your help and the wonderful addon.

Ian Robson
Bytelink LLC.
5616 Devon Street
Port Orange, FL 32127
(386) 235-9563  Cell
